collector/src/test/java/com/google/cloud/trace/zipkin/PlainTextNettyChannelProvider.java

```diff
@@ -0,0 +1,37 @@
+package com.google.cloud.trace.zipkin;
+
+import com.google.auto.service.AutoService;
+import io.grpc.ManagedChannelProvider;
+import io.grpc.netty.NettyChannelBuilder;
+
+/**
+ * Allows non-ssl grpc connections for {@link NettyChannelBuilder} instances.
+ */
+@AutoService(ManagedChannelProvider.class)
+public class PlainTextNettyChannelProvider extends ManagedChannelProvider
+{
+  @Override
+  public boolean isAvailable()
+  {
+    return true;
+  }
+
+  @Override
+  public int priority()
+  {
+    //put it in front of all other providers: NettyChannelProvider/OkHttpChannelProvider
```

Comment:
@kevinmdavis mock TraceSink is where test started, but gradually moved to full fledge integration test with only one mock left: GoogleCredentials.

I like the idea of adding new constructor to Sink that accepts just TraceServiceBlockingStub, meanwhile I can remove `PlainTextNettyChannelProvider`  and replace it with "hacky" Reflection solution while upstream SDK is being patched:
```
    @Bean(name = "traceSink")
    @Primary
    TraceSink traceSink(Credentials credentials, ZipkinStackdriverStorageProperties storageProperties)
        throws IOException, NoSuchFieldException, IllegalAccessException
    {
      final GrpcTraceSink traceSink = new GrpcTraceSink(storageProperties.getApiHost(), credentials);
      final ManagedChannel managedChannel = ManagedChannelBuilder.forTarget(storageProperties.getApiHost()).usePlaintext(true).build();
      TraceServiceGrpc.TraceServiceBlockingStub traceService = TraceServiceGrpc.newBlockingStub(managedChannel)
          .withCallCredentials(MoreCallCredentials.from(credentials));

      final Field traceServiceField = GrpcTraceSink.class.getDeclaredField("traceService");
      traceServiceField.setAccessible(true);
      traceServiceField.set(traceSink, traceService);

      return traceSink;
    }

```